src/common/common.vcxproj

```diff
@@ -82,7 +82,8 @@
       <GenerateDebugInformation>true</GenerateDebugInformation>
     </Link>
     <Lib>
-      <AdditionalDependencies>shlwapi.lib;shcore.lib</AdditionalDependencies>
+      <AdditionalDependencies>
+      </AdditionalDependencies>

```

Comment:
The changes to this file go along with the change in `src/common/dpi_aware.h` correct?
Let's have one PR for this.

Reply:
opened https://github.com/microsoft/PowerToys/pull/5930